MantidQt/MantidWidgets/inc/MantidQtMantidWidgets/DataProcessorUI/QDataProcessorWidget.h

```diff
@@ -86,7 +86,7 @@ class EXPORT_OPT_MANTIDQT_MANTIDWIDGETS QDataProcessorWidget
                             const std::string &defaultValue) override;
   bool askUserYesNo(std::string prompt, std::string title) override;
   void giveUserWarning(std::string prompt, std::string title) override;
-  void giveUserCritical(std::string prompt, std::string title) override;
+  void giveUserCritical(QString prompt, std::string title) override;
```

Comment:
It seems a little inconsistent to me that we accept QString for one argument but the other is a `std::string`.

I think we should take the opportunity to make this consistent across this class. As I assume most of the time these interact with Qt widgets then it feels like we should use `QString` across the class. @AntonPiccardoSelg Any thoughts here?